vault/core.go

```diff
@@ -2457,7 +2457,8 @@ func (s standardUnsealStrategy) unseal(ctx context.Context, logger log.Logger, c
 		c.StartManualCensusSnapshots()
 
 	} else {
-		broker, err := audit.NewBroker(logger)
+		brokerLogger := logger.Named("audit")
```

Comment:
Would `auditbroker` be even more specific? I'm not sure if it's better but I thought I'd raise it as a thought!